kitsune/search/v2/search.py

```diff
@@ -86,11 +86,11 @@ def make_result(self, hit):
         # generate a summary for search:
         summary = first_highlight(hit)
         if not summary:
-            summary = hit.question_content[self.locale][:SNIPPET_LENGTH]
+            summary = getattr(hit.question_content, self.locale, "")[:SNIPPET_LENGTH]
```

Comment:
If a question hit lacks a `question_content` in the appropriate locale, it should also lack a title. Wiki docs are also susceptible to this.

I don't think we should make a change like this, because it fundamentally doesn't make sense in the context of search: we only search over `question_content.<locale>`, so if a result lacks content in `question_content.<locale>` it means the user has done a unconventional search, which won't return them anything useful anyway, like `*` or `-foobar`.

I think we should just catch a keyerror like this and return an empty result set, something like (in `base.py`):

```
try:
    self.results = [self.make_result(hit) for hit in self.hits]
except KeyError as e:
    if e.message == self.locale:
        self.results = []
        self.total = 0
    else:
        raise e
```